routes/page-routes.js

```diff
@@ -9,7 +9,7 @@ module.exports = (app) => {
     res.status(200).render('about');
   });
   // route to display a list of the trending images
-  app.get('/trending', ({ params, headers }, res) => {
+  app.get('/popular', ({ params, headers }, res) => {
```

Comment:
The old URL ought to be 301 redirect to the new one.

Reply:
added redirect https://github.com/lbryio/spee.ch/commit/7a81d53c3570d07e8666aee2fbfd7209e7646827